src/tests/nunit.runner.tests.Droid/MainActivity.cs

```diff
@@ -25,6 +25,8 @@
 using Android.Content.PM;
 using Android.OS;
 
+using NUnit.Runner.Services;
```

Comment:
Changes to each of these platform specific files also need to be made to the versions that are in the NuGet packages, for example https://github.com/nunit/nunit.xamarin/blob/master/nuget/MonoAndroid10/MainActivity.cs.txt.pp

In the future, it would be nice to pull those files from these and do the transform on the namespace.